### ISSUE TITLE
Bug when 1st entering asteroid dimension 

### ISSUE THREAD

**Issue by dwaggner** [micdoodle8/Galacticraft#643]
Hey guys, Daniel here, and today I was playing galacticraft 1.7.2-3.0.0.129. and I noticed that when ever I go to the asteroid dimension, there is an issue with the newly implemented lander.
When gc3 beta was first released, the 1st thing I did was (after backing up my worlds of course) head to the asteroid dimension. I have a crappy pc and when ever I load up a new dimension from galacticraft (as in generate it for the 1st time on that world), it usually fall from 100 blocks above the void, and stop falling at around -0.5 blocks from bedrock, then i'll stop falling and have to wait for the dimension to load up, then, finally, proceed down to the surface of the planet in the manner the specified dimension allows it. 
example:
Moon:
fall from +100 to -0.5
load chunks
go through the processes the lander has (press space and land) 

This was very different however.
When I entered the dimension, i was falling without the lander. I thought this was normal untill I saw an asteroid below me generate, and have me land on it. I then thought that was simply because the lander hadn't been implemented in that version of galacticraft 3. 

I then decided to watch some Skydoesminecraft because he usually uploads a video or 2 every day. 

A few hours later, I rejoined that dimension, and I spawned, again, in the air, but this time, I was in the landing pod. I thought that this ocerred because the world itself was bugged.  

I then proceeded to create a new world, hoping that this issue was simply from the old world being bugged. The issue was still there. I then restarted the world, and I was in the landing pod. 

Basically, what i'm trying to say is that when ever somebody enters the asteroid dimension for the 1st time, you will fall untill you hit an asteroid, and then you will fall in the entry pod when you reload the world. 

Also, 1 last thing I should note: you cant place and blocks from what I can see (it wont be there when you reload the world), you cant drop any items, and you can't use any items.

Thank you for reading this bug
-Daniel.
